ui/src/wide_series_stat.tsx

```diff
@@ -28,12 +28,12 @@ const
       padding: grid.gap,
     },
     plot: {
-      display: 'flex',
-      width: pc(50),
+      position: 'relative',
```

Comment:
What is this position for? IIRC plots are not positioned absolutely or maybe I am missing something?

Reply:
I've just added it for implementation consistency with wide gauge stat, but now I see this was not necessary.